docs/CONFIG-VARS.md

```diff
@@ -279,7 +279,7 @@ V4_CFG_POSTGRES_SERVERS:
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| CERT_MANAGER_ENABLED | Whether to deploy tool | bool | true | false | | baseline |
+| CERT_MANAGER_ENABLED | Whether to deploy tool | bool | false | false | | baseline |
```

Comment:
Just a doc comment.
Since there are now two certificate handling objects that IAC can impact, I think it might help to say, "Whether to deploy certificate-manager into the cluster" to be more specific than "tool" here.

Reply:
@dhoucgitter, you're right that description doesn't seem to be clear. I've updated the description and notes section to:

| Name | Description | Type | Default | Required | Notes | Tasks |
| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
| CERT_MANAGER_ENABLED | Whether to deploy cert-manager into the cluster using helm | bool | false | false | Required if V4_CFG_TLS_GENERATOR is set to `cert-manager` and it's not already installed | baseline |

Let me know if that sounds better